test/release/examples/primers/domains.chpl

```diff
@@ -1,6 +1,6 @@
 // Domains
 
-/*
+/* BLAHH 5000!
```

Comment:
Que?  That shows up in the rendered doc

https://chapel-lang.org/docs/main/primers/domains.html

Reply:
... I'm not sure how I missed that

Reply:
I only spotted it because with two haitches it didn't spellcheck.

Reply:
🤦‍♂️ cruft I added for testing the doc changes. I'll remove it. Thanks!

Reply:
PR with the "fix" here: https://github.com/chapel-lang/chapel/pull/18897/